content-security-policy/connect-src/connect-src-json-import-allowed.sub.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>connect-src-json-import-allowed</title>
+    <meta
+      http-equiv="Content-Security-Policy"
+      content="connect-src 'self' http://{{host}}:{{ports[http][0]}}; script-src 'self' 'unsafe-inline';"
+    />
+    <script src="/resources/testharness.js"></script>
+    <script src="/resources/testharnessreport.js"></script>
+  </head>
+
+  <body>
+    <script>
+      promise_test(async (t) => {
+        window.addEventListener(
+          "securitypolicyviolation",
+          t.unreached_func("No security policy violation should be raised.")
```

Comment:
Can we make sure that this is the right securitypolicyviolation? Maybe by checking that blockedURI ends with /common/dummy.json?

I know we had flakiness problems in the past because of mismatching violations (for example, I remember a problem around favicons)

Reply:
This is asserting that it's _not_ blocking it (t.unreached_func asserts that the function is not called). I can update it to test that, if the event is fired, it's not fired for the allowed resource.

Reply:
Yes, I understand that. What I'd like to avoid is that this test becomes flaky because of an unrelated security policy violation firing.